website/generate.py

```diff
@@ -137,6 +147,13 @@ def fetch_report_data(report_dir):
 report_template = env.get_template('report.html.jinja')
 
 for year, month, entry in iter_report_entries(index_data["reports"]):
+
+    # DEV ONLY, PLEASE REMOVE ME ***************************
```

Comment:
Let's remove this for pulling in the PR.

Reply:
Done